backend/rebutify/settings.py

```diff
@@ -211,7 +214,8 @@
 EMAIL_PORT = int(os.getenv("EMAIL_PORT", 587))
 EMAIL_USE_TLS = bool(os.getenv("EMAIL_USE_TLS", True))
 
-EMAIL_FROM = os.getenv("EMAIL_FROM", "")
+EMAIL_FROM = os.getenv("EMAIL_FROM", "noreply@rebutify.org")
```

Comment:
Do we really need to set something other than noreply@rebutify.org ?

Reply:
Well the project is open source, so anyone can host their own instance. In which case they'd need something different.